internal/artifacts/module_arch.go

```diff
@@ -142,12 +142,15 @@ func packModule(ep dir.IModule, deploymentDesc bool, module *mta.Module, moduleN
 			moduleName)
 	}
 
-	entry, err := os.Stat(buildResults)
-	if err != nil {
-		return errors.Wrapf(err, `packing the "%v" module failed; the "%v" build results path does not exist`,
-			moduleName, buildResults)
+	definedArchive := false
+	if buildResults != "" {
+		definedArchive, err = isArchive(buildResults)
+		if err != nil {
+			return errors.Wrapf(err, `could not find the "%s" build results`, buildResults)
```

Comment:
Shouldn't this be the same error message as before? (packing the "%v" module failed; the "%v" build results path does not exist) Or did you change it because the error doesn't necessarily mean it doesn't exist? If so I suggest "could not read the build results" instead of "could not find", and add the first part with "packing the module failed"